src/Formidable.res

```diff
@@ -316,16 +316,14 @@ module Make = (ValidationLabel: Type, Error: Type, Values: Values): (
 
       let validationNames =
         validations->Option.mapWithDefault([], validations =>
-          validations->ArrayExtra.flatMap(validation =>
-            validation->Validations.getNames->List.toArray
-          )
+          validations->ArrayExtra.flatMap(validation => validation->Validations.getNames)
```

Comment:
could be shorten to just do `flatMap(Validations.getNames)` right Kevin-san

Reply:
absolutely right I dropped the function call but didn't update the code, let me fix that 👍